fdbserver/TenantBalancer.actor.cpp

```diff
@@ -1680,6 +1680,60 @@ ACTOR Future<Void> recoverMovement(TenantBalancer* self, RecoverMovementRequest
 	return Void();
 }
 
+ACTOR Future<Void> clearTenant(TenantBalancer* self,
+                               Reference<MovementRecordMap::MutableRecord> mutableRecord,
+                               MovementLocation location,
+                               bool doErase,
+                               bool doUnlock) {
+	state Reference<MovementRecord> record = mutableRecord->record;
+	state Key targetPrefix =
+	    location == MovementLocation::SOURCE ? record->getSourcePrefix() : record->getDestinationPrefix();
+	if (doErase) {
+		// Erase the moved data, if desired
+		KeyRange rangeToErase = prefixRange(targetPrefix);
+		wait(waitOrError(runTenantBalancerTransaction<Void>(self->db,
+		                                                    self->tbi.id(),
+		                                                    "Clea drTenantErase",
```

Comment:
Something weird has happened here this time

```suggestion
		                                                    "ClearTenantErase",
```

Reply:
Solved. Sorry for the weird case.